packages/next-tinacms-cloudinary/src/handlers.ts

```diff
@@ -106,7 +107,7 @@ async function listMedia(
 
     const response = await cloudinary.search
       .expression(query)
-      .max_results(limit)
+      .max_results(parseInt(limit as string, 10))
```

Comment:
If we change back `limit` to a number do we still need to parseInt?

Reply:
Yeah we wouldn't need to. I'll spin up a better type definition above